docs/nodes/GENERATORS/SIMULATIONS/MATRIX/MATRIX.md

```diff
@@ -0,0 +1,57 @@
+
+[//]: # (Custom component imports)
+
+import DocString from '@site/src/components/DocString';
+import PythonCode from '@site/src/components/PythonCode';
+import Parameters from '@site/src/components/Parameters';
+import AppDisplay from '@site/src/components/AppDisplay';
+import SectionBreak from '@site/src/components/SectionBreak';
+import AppendixSection from '@site/src/components/AppendixSection';
+
+[//]: # (TODO: Machine-generate this section)
+
+import DocstringSource from '!!raw-loader!./a1-[autogen]/docstring.txt';
+import PythonSource from '!!raw-loader!./a1-[autogen]/python_code.txt';
+import ParametersSource from '!!raw-loader!./a1-[autogen]/parameters.yaml';
+
+<DocString>{DocstringSource}</DocString>
+<PythonCode GLink='GENERATORS/SIMULATIONS/MATRIX/MATRIX.py'>{PythonSource}</PythonCode>
+<Parameters>{ParametersSource}</Parameters>
+
+<SectionBreak />
+
+    
+
+[//]: # (Examples)
+
+## Examples
+
+import Example1 from './examples/EX1/example.md';
+import App1 from '!!raw-loader!./examples/EX1/app.txt';
+import Data1 from '!!raw-loader!./examples/EX1/output.txt';
+
+<AppDisplay 
+  GLink='GENERATORS/SIMULATIONS/MATRIX'
```

Comment:
This prop is only given when there is no example available to help users to navigate to the node function file.

Reply:
`data` props must be provided for `<AppDisplay>` when an example is available